pyscriptjs/tests/integration/test_00_support.py

```diff
@@ -32,6 +32,17 @@ def test_basic(self):
         content = self.page.content()
         assert "<h1>Hello world</h1>" in content
 
+    def test_no_cors_headers(self):
+        self.disable_cors_headers()
+        doc = """
+        <html>
+          <script>console.log(typeof SharedArrayBuffer === "undefined")</script>
```

Comment:
I guess it's better just to use [crossOriginIsolated](https://developer.mozilla.org/en-US/docs/Web/API/crossOriginIsolated)

Reply:
done